Samples/YOLOv4ObjectDetection/YOLOv4ObjectDetection/MainPage.xaml.cs

```diff
@@ -0,0 +1,372 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Windows.AI.MachineLearning;
+using Windows.Devices.Enumeration;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Media;
+using Windows.Media.Capture;
+using Windows.Storage;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using Windows.UI.Xaml.Navigation;
+using Windows.UI.Xaml.Shapes;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace YOLOv4ObjectDetection

```

Comment:
I think this is based on the AI skills implementation, right? Can you check with Alex if we should link to their repo and mention that it's based on that implementation?

Reply:
This is not depend on any AI kills implemetation as I can see. I just renamed the namespace to align StyleTransferObjectDetection in our Repo. Where else do you think it might be related to AI skills?

---
In reply to: [568061428](https://github.com/microsoft/Windows-Machine-Learning/pull/376#discussion_r568061428) [](ancestors = 568061428)